admin/class-automation.php

```diff
@@ -103,15 +103,17 @@ public static function action__admin_menu(): void {
 	 * A callback function for the apple_news_active_theme filter.
 	 *
 	 * @param string $theme_name The name of the theme to use.
-	 * @param int    $post_id    The ID of the post being exported.
+	 * @param ?int   $post_id    The ID of the post being exported.
 	 *
 	 * @return string The filtered theme name.
 	 */
 	public static function filter__apple_news_active_theme( $theme_name, $post_id ) {
-		$rules = self::get_automation_for_post( $post_id );
-		foreach ( $rules as $rule ) {
-			if ( 'theme' === ( $rule['field'] ?? '' ) && ! empty( $rule['value'] ) ) {
-				return $rule['value'];
+		if ( $post_id ) {
+			$rules = self::get_automation_for_post( $post_id );
+			foreach ( $rules as $rule ) {
+				if ( 'theme' === ( $rule['field'] ?? '' ) && ! empty( $rule['value'] ) ) {
+					return $rule['value'];
+				}
 			}
 		}
```

Comment:
Fix for the theme/automation null condition default edge case.  Entire function returns `$theme_name` (`Default`) if `$post_id` null, not visible in the git diff.